central/detection/service/service.go

```diff
@@ -49,6 +50,7 @@ func New(
 		clusters:           clusters,
 		imageEnricher:      imageEnricher,
 		imageDatastore:     imageDatastore,
+		imageEnricherV2:    imageEnricherV2,
```

Comment:
nit: why not put this next to the other `imageEnricher`?

(same for the other instances)

Reply:
Looking again, I realized that some struct definitions had enricherV2 after enricherV1, but others had a different order. Updated it to make it consistent in all places.